tensorflow/core/kernels/mkl/mkl_matmul_ops_common.h

```diff
@@ -40,7 +40,8 @@ namespace tensorflow {
 static Eigen::internal::CacheSizes cache_sizes = Eigen::internal::CacheSizes();
 
 typedef Eigen::ThreadPoolDevice CPUDevice;
-inline bool ExecuteSingleThreadedGemm(int m, int n, int k, int bytes) {
+inline bool ExecuteSingleThreadedGemm(int64_t m, int64_t n, int64_t k,
+                                      int bytes) {
```

Comment:
Maybe we should add `mul_size >= 0 && ` to the condition? 
https://github.com/tensorflow/tensorflow/blob/1448daa62f02cbf87412ab55c32ae762c08a2ccb/tensorflow/core/kernels/mkl/mkl_matmul_ops_common.h#L54

We can do it in a follow-up PR if it will take time to test this.

Reply:
@penpornk Thanks! I am actually doing the follow-up PR which addresses all dim size related code in oneDNN code base. I will take care of your suggestion